sourcegraph/about
# Audit & Add missing languages for highlighting

Closes #5521 by adding missing languages to code blocks for the syntax highlighter to pick up
### Changelog
""

### Note
Supported Prism languages can be found [here](https://prismjs.com/#supported-languages)

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure the changed blog posts code blocks are now highlighted properly.
